backend/app/services/archive_enrollments.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class ArchiveEnrollments
```

Comment:
Why did you split in 2 classes ? There is 2 options:

1. You need to make a disjonction: you can pass an argument (`statuses`) to the class, and pass it to your sql query
2. You don't need to make a disjonction: just use all statuses you want.

Reply:
Should I rename this class `EnrollmentsArchiveable` ?

Reply:
yes